app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,145 @@
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>
+quote  </p>
+
+  <p>
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your pupils' ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover include:
+      </p>
+      <p>
+         Themes you’ll cover for 11 to 14 year olds (key stage 3):
+      </p>
+        <ul class="vertical-tags">
+          <li class="tag">Energy</li>
+          <li class="tag">Motion and forces</li>
+          <li class="tag">Electricity and electromagnetism</li>
+          <li class="tag">Waves and matter</li>
+        </ul>
+      <p>
+        Themes you’ll cover for 14 to 16 year olds (key stage 4):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+      <p>
+        Themes you’ll cover for 16 to 18 year olds (key stage 5):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search for past exam papers online to see the type of questions that were asked in previous years.</p>
+
+    <p>
+      You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </p>
+    <p><a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">Find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics. </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "Next steps into teacher training",
```

Comment:
@caitlin-reidy This page has a next steps into teacher training header. Do you think we should have this on all of the pages or remove it from this one so they're all consistent?